bokeh/models/widgets/markups.py

```diff
@@ -28,6 +28,26 @@ class Paragraph(Markup):
     The height of the block in pixels.
     """)
 
+class Div(Markup):
+    """ A block (div) of text.
+
+    """
+
+    text = String(default="", help="""
+    The contents of the widget.
+    """)
+
+    width = Int(500, help="""
+    The width of the block in pixels.
+    """)
+
+    height = Int(400, help="""
```

Comment:
Large = SLOC?
Large = Height?

This could be dramatically dropped down if these defaults are pulled into the Markup class and the Paragraph and the Div are simple subclasses with the whole intent on just changing the tag type.  This was what I was getting at in my original comments.  There is a lot of code duplication.  I was just trying to get something present which could be cleaned up later.